boards/nucleo-f334/dist/openocd.cfg

```diff
@@ -0,0 +1,6 @@
+# nucloe-f334 Target
+source [find interface/stlink-v2-1.cfg]
+
+transport select hla_swd
+
```

Comment:
You can discard this file. It's the same content as in `board/st_nucleo_f334r8.cfg`